### DISCUSSION TITLE
npm publish returns "ok" but does not actually publish anything

### DISCUSSION THREAD

**Discussion by jorroll** [npm/feedback#580] [Registry]
I appear to be running into [npm/npm#6308](https://github.com/npm/npm/issues/6308) with one of my packages, [rx-controls-react](https://www.npmjs.com/package/rx-controls-react).

npm CLI (via `yarn publish`) reports that I've successfully published version `0.8.6` of `rx-controls-react`. When I navigate to  npmjs.com and [view the package](https://www.npmjs.com/package/rx-controls-react) however, it reports `0.8.5` as the latest version. Attempting to install version `0.8.6` of `rx-controls-react` in a separate project fails with a message that 

```
Couldn't find any versions for "rx-controls-react" that matches "^0.8.6"
```

It's been about 10-15 minutes since I published version `0.8.6` of the package. Attempting to re-publish version `0.8.6` of the package results in the following error:

```
error Couldn't publish package: "https://registry.yarnpkg.com/rx-controls-react: You cannot publish over the previously published versions: 0.8.6."
```

Apologies if this is really a `yarn cli` error, but it seems most likely to be an npm registry issue (given that this issue [appears to have happened in the past](https://github.com/npm/npm/issues/6308)).

I'll note that, at the same time that I published version `0.8.6` of `rx-controls-react`, I also published version `0.8.6` of `rx-controls`, `rx-controls-solid`, and `rx-controls-angular`. Those three packages were successfully published. Only `rx-controls-react` has had an issue.

**Comment by jorroll** [comment]
To be safe, I just tried publishing the package again using the npm cli and I ran into the following error:

```
npm ERR! 403 403 Forbidden - PUT https://registry.npmjs.org/rx-controls-react - You cannot publish over the previously published versions: 0.8.6.
```

**Comment by ljharb** [comment]
See https://status.npmjs.org/ - there’s an ongoing incident.
